assume/world.py

```diff
@@ -609,9 +623,11 @@ async def run_simulation(self):
                 self.clock.set_time(end_ts)
 
             await tasks_complete_or_sleeping(self.container)
-        pbar.close()
+        # pbar.close()
```

Comment:
This is actually correct to close the context here (just like you would close a open file handle) - but as the program terminates anymore, the context is closed there too - still I would revert this line?